app/views/list/show.html.slim

```diff
@@ -0,0 +1,132 @@
+html[lang="en"]
+  head
+    meta[charset="utf-8"]
+    title
+      | Wishlist
+  body
+    .container-fluid
+      .row
+        .col-xs-12
+          .wishlist-header
+            | Wishlist
+          .wishlist-page
+            .table-responsive
+              table#catalog-table.wishlist-table.table.table-hover
+                thead
+                  tr
+                    th
+                    th
+                    th
+                      | Request
+                    th
+                      | Recipient
+                    th
+                      | Status
+                    th
+                      | Age
+                    th
+                      | Gender
+                    th
+                      | Description
+                tbody
+                  tr.request
```

Comment:
Should also be pulled out into a partial (even though this is boilerplate for now - you should be providing boilerplate data to the partial while testing).